packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -531,6 +531,12 @@ IWorker engine = WorkerFactory.CreateWorker(BackendType.GPUCompute, model);
 `,
 ];
 
+export const voicecraft = (model: ModelData): string[] => [
+	`from voicecraft import VoiceCraftHF
+
+model = VoiceCraftHF.from_pretrained("${model.id}")`,
```

Comment:
they didn't want to add it to their main `VoiceCraft` class?

Reply:
No it's because of a limitation of the Mixin class, I had to define a separate `VoiceCraftHF` class in order for it to work. This is because the original class takes a `argparse.Namespace` as keyword argument, which is not supported by the Mixin (cc @Wauplin)

Reply:
I do think we should be more pushy to make small changes on the external libs when integrating the mixin class. In particular we should aim at always having a single class. The problem of having 2 classes is that if a user instantiates the base class in a training script, they cannot push it to the Hub  at the end of the training. This is a bad developer experience which defeats the goal of the mixin.

I modified PR https://github.com/jasonppy/VoiceCraft/pull/90 and especially https://github.com/jasonppy/VoiceCraft/commit/e550f614096a9ba098d2127a4e441ca497ba256f in that sense.

Reply:
Also opened https://github.com/huggingface/huggingface.js/pull/630.